docs/src/development/local/ddev.md

```diff
@@ -84,16 +78,17 @@ ddev config --web-environment-add="PLATFORM_PROJECT={{< variable "PROJECT_ID" >}
 For example:
 
 ```bash
-ddev config --web-environment-add="PLATFORM_PROJECT=nf36mudfdd23bi,PLATFORM_ENVIRONMENT=main"
+ddev config --web-environment-add="PLATFORM_PROJECT=abcdefgh1234567,PLATFORM_ENVIRONMENT=main"
 ```
 
-Get DDEV to recognize the variables by restarting:
+The best way to connect your local DDEV to your Platform.sh project is through the [Platform.sh DDEV add-on](https://github.com/platformsh/ddev-platformsh).
+Add it by running the following command:
 
 ```bash
-ddev restart
+ddev get platformsh/ddev-platformsh
```

Comment:
The add-on will soon become "official", so the command will slightly be simplified.

```suggestion
ddev get ddev-platformsh
ddev restart
```

Reply:
It has now moved to drud/ddev-platformsh.